articles/welcome.md

```diff
@@ -10,48 +10,71 @@ ms.topic: overview
 
 # Get started with the Quantum Development Kit (QDK)
 
-Welcome to the Microsoft Quantum Development Kit!  Here, you will find all the tools you need to learn about quantum programming with Q#.  To guide you in learning quantum computing with Q#, we provide this Getting Started Guide, a journey guide for both those of you who want to start coding quantum programs, and people who are not yet ready to start coding but want to learn more about Q# and quantum programming.
+Welcome to the Microsoft Quantum Development Kit!  
+The QDK contains all the tools you'll need to build your own quantum computing programs and experiments with Q#, a programming language used for expressing quantum algorithms. 
+Specifically, Q# allows you to write sub-programs that execute on an adjunct quantum processor under the control of a classical host program and computer. 
```

Comment:
I'm not sure this line is necessary

Reply:
Yeah I was on the fence about it as well. Figured it might serve to clarify for anyone wondering what the "point" of Q# is alongside classical programs, but this early I think it'd just create more confusion than it alleviates. Will cut it.